target/linux/octeon/base-files/lib/preinit/79_move_config

```diff
@@ -1,19 +1,23 @@
 # Copyright (C) 2014 OpenWrt.org
 
-move_config() {
-	. /lib/functions.sh
+octeon_move_config() {
 	. /lib/upgrade/common.sh
+	local device="$1"
+	[ -n "$device" ] && [ -b "$device" ] && {
+		mount -t vfat "$device" /mnt
+		[ -f "/mnt/$BACKUP_FILE" ] && mv -f "/mnt/$BACKUP_FILE" /
+		umount /mnt
+	}
+}
 
+move_config() {
+	. /lib/functions.sh
 	case "$(board_name)" in
 		erlite)
-			mount -t vfat /dev/sda1 /mnt
-			[ -f "/mnt/$BACKUP_FILE" ] && mv -f "/mnt/$BACKUP_FILE" /
-			umount /mnt
+			octeon_move_config "/dev/sda1"
```

Comment:
To be honest, I'd swap the names for the functions. The current move_config actually does octeon-specific stuff, while the current octeon_move_config does generic stuff.

Reply:
sure, thank you for the feedback. swapped the names.